.github/workflows/distribute-grain.yml

```diff
@@ -44,17 +46,13 @@ jobs:
           echo "::set-output name=pr_body::$description"
           rm grain_output.txt
 
-      - name: Create commit and PR for ledger changes
-        id: pr
-        uses: peter-evans/create-pull-request@v3
-        with:
-          branch: generated-ledger
-          branch-suffix: timestamp
-          committer: credbot <credbot@users.noreply.github.com>
-          # author appears to be overridden when the default github action
-          # token is used for checkout
-          author: credbot <credbot@users.noreply.github.com>
-          commit-message: update calculated ledger
-          title: ${{ env.PULL_REQUEST_TITLE }}
-          body: ${{ steps.pr_details.outputs.pr_body }}
-          reviewers: decentralion
+      - name: Commit ledger changes
+        run: |
+          git config user.name 'credbot'
+          git config user.email 'credbot@users.noreply.github.com'
+          git add data/ledger.json
+          git commit --allow-empty -m '${{ env.PULL_REQUEST_TITLE }}' -m '${{ steps.pr_details.outputs.pr_body }}'
```

Comment:
why do we use the `-m` flag twice?

Reply:
Using two `-m` flags and arguments yields the `pr_details` on its own line, leaving an empty line between the top line of the commit and the details. 

This is basically an easy way to create a properly formatted and detailed commit message without messing with any further formatting.